src/Span.h

```diff
@@ -730,6 +733,41 @@ class Span
         return *this;
     }
 
+    /**
+    * \brief Report the span is sharing between the client and the server.
+    */
+    inline bool shared(void) const { return m_shared; }
```

Comment:
so this would be set when you successfully extract a trace context from headers (and use it)